app/assets/javascripts/settings.coffee

```diff
@@ -11,9 +13,26 @@ parseFloatOrElse = (str, def) ->
 clamp = (min, max, val) ->
   Math.max(min, Math.min(max, val))
 
+updatedLocales = locales.map((locale) ->
```

Comment:
With the hard-coded `languageCode` values in `locales`, I don't think this clone is necessary.  `updatedLocales` can be removed.